File: Beecrowd/3048 Sequência Secreta/main.cpp
#include <iostream>

using namespace std;

int main(){
    int n, cont = 1;

    cin >> n;

    int vet[n];

    for(int i = 0; i < n; i++){
        cin >> vet[i];
    }

    for(int i = 1; i < n; i++){
        if(vet[i] != vet[i - 1])
            cont++;
    }

    cout << cont << endl;

    return 0;
}

File: Beecrowd/3048 Sequência Secreta/Main.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args)  throws Exception{
        Scanner sc = new Scanner(System.in);

        int n = sc.nextInt();
        int cont = 1;
        int vet[] = new int[n];

        for(int i = 0; i < n; i++){
            vet[i] = sc.nextInt();
        }

        for(int i = 1; i < n; i++){
            if(vet[i] != vet[i - 1])
                cont++;
        }

        System.out.println(cont);

        sc.close();
    }
}